examples/minimal-winit-android/src/lib.rs

```diff
@@ -0,0 +1,132 @@
+#[cfg(target_os = "android")]
+use winit::platform::android::activity::AndroidApp;
+
+use pixels::{Pixels, SurfaceTexture};
+use winit::event::Event;
+use winit::event_loop::{ControlFlow, EventLoop, EventLoopBuilder};
+use winit::window::Window;
+
+const WIDTH: u32 = 320;
+const HEIGHT: u32 = 240;
+const BOX_SIZE: i16 = 64;
+
+/// Representation of the application state. In this example, a box will bounce around the screen.
+struct World {
+    box_x: i16,
+    box_y: i16,
+    velocity_x: i16,
+    velocity_y: i16,
+}
+
+fn _main(event_loop: EventLoop<()>) {
+    let mut window: Option<Window> = None;
+    let mut pixels: Option<Pixels> = None;
+
+    let mut world = World::new();
+
+    event_loop.run(move |event, event_loop, control_flow| {
+        *control_flow = ControlFlow::Wait;
+        match event {
+            Event::Resumed => {
+                let _window = Window::new(&event_loop).unwrap();
+                let _pixels = {
+                    let window_size = _window.inner_size();
+                    let surface_texture =
+                        SurfaceTexture::new(window_size.width, window_size.height, &_window);
+                    Pixels::new(WIDTH, HEIGHT, surface_texture).unwrap()
+                };
+                _window.request_redraw();
+                window = Some(_window);
+                pixels = Some(_pixels);
+            }
+            Event::Suspended => {
+                window = None;
+                pixels = None;
```

Comment:
Technically, `pixels` should be dropped before `window`.

The reason is subtle, but you can read more about it in #238. In short, `Pixels` owns a `SurfaceTexture` which borrows the `Window` [^1]. Therefore the `Window` should outlive the `Pixels`.

It's unlikely to cause problems in this case (since both are dropped in close proximity and they aren't shared by other threads or anything), so it's just a technicality.

[^1]: The normal way to declare this kind of dependency in Rust types is with a lifetime, but that would have been too restrictive. The linked ticket has more info, and some thoughts on how this API issue could be fixed.